src/components/TagsDropDown/index.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import TAGS from '../../constants/tags';
+import './index.scss';
+
+const TagsDropDown = ({ onTagsSelected }) => {
+  const { tick } = useSelector(({ posts }) => posts);
+
+  const [showTags, setShowTags] = useState();
```

Comment:
Are we mixing hooks with class component in the same code base?

Reply:
@oesukam I find it better to write new components in hooks and progressively refactor existing components too, as hooks reduce complexity.

Reply:
OK. You can provide a default value instead of having it as null